irc3/plugins/command.py

```diff
@@ -268,7 +268,7 @@ def split_command(self, data, use_shlex=True):
             if use_shlex:
                 return shlex.split(data)
             else:
-                return data.split()
+                return data.split(' ')
```

Comment:
The problem is that it's not the same result when you have more than one space:

```py
>>> ' a  a'.split()
['a', 'a']

>>> ' a  a'.split(' ')
['', 'a', '', 'a']
```

I like the idea to allow and ignores extra spaces in commands (typo tolerance).

May be it can work if you ignore empty strings in the splited results